INSTALL.rst

```diff
@@ -175,4 +239,33 @@ entry points and then do the following (e.g. for the `hep` rule):
 
 .. code-block:: console
 
-    dojson -l marcxml -i inspirehep/demosite/data/sample.xml do hep
+    (inspirehep)$ dojson -l marcxml -i inspirehep/demosite/data/sample.xml do hep
+
+
+Giving your user admin access
+==============================
+
+If you want access to the admin console, you'll have to execute:
+
+.. code-block:: console
+
+    (inspirehep)$ inspirehep access allow -e your@email.com admin-access
+
+And for the holding pen page admin access:
+
+.. code-block:: console
+
+    (inspirehep)$ inspirehep access allow -e your@email.com worflows-ui-admin-access
+
+.. note::
+
+    This feature is still unstable, and the name of the authorization access
+    might change, if you are having problems with it, you can check all the
+    access actions with:
+
+    .. code-block:: console
+
+        (inspirehep)$ inspirehep access list
+
+.. _this issue: https://github.com/inspirehep/inspire-next/issues/1296
```

Comment:
They do, just only on pythonhosted (or readthedocs). It's sphinx (restructured text) syntax, and github does not like it, I can try to investigate if it's possible to please both systems in any way